src/js/actions/pluginSettings.js

```diff
@@ -68,11 +68,19 @@ export function asyncPluginFetchSettings(zoneId) {
 export function asyncPluginUpdateSetting(settingName, zoneId, value) {
     return (dispatch, getState) => {
         let oldSetting = getState().pluginSettings.entities[zoneId][settingName];
+
+        if (settingName == "default_settings") {
```

Comment:
`ApplyDefaultSettingsCard.js` calls `dispatch(asyncPluginUpdateSetting(SETTING_NAME, activeZoneId, VALUE));`

Is there an easy way to catch the result of `asyncPluginUpdateSetting` inside `ApplyDefaultSettingsCard`?